client/src/components/Home.vue

```diff
@@ -3,9 +3,14 @@
     h1 Home
     ul(v-if="results")
       li(v-for="item in results.results", v-on:click="goToPlaylist(item)")
-        img(v-if="item.images[0]", :src="item.images[0].url")
-        div(v-else, class="no-image") No image found
-        div(class="name") {{ item.name }}
+        v-container(grid-list-md)
+          v-layout(row wrap)
+            v-flex
```

Comment:
Let's make it responsive. Something like `v-flex(xl2, lg3, md4, sm6, xs12)` should do the trick.